### ISSUE TITLE
Mobile display unsopported

### ISSUE THREAD

**Issue by TruddyTheDuddi** [TruddyTheDuddi/loquacious#1]
Mobile browser unsupported for now.
I will be working on the mobile adaptation for each page after I'm completely done with the desktop version of a page.

**Comment by hacknorris-notyetbanned** [comment]
W00t ? Now i use ios andit's just good... I've even made a few posts...

**Comment by TruddyTheDuddi** [comment]
Oh I see. This repository is a different project from what you're seeing on https://loquacious.xyz
If you want to see a preview of this "previous" loquacious project, you can check it out here: https://truddytheduddi.github.io/loquacious/

Enjoy! :)

**Comment by hacknorris-notyetbanned** [comment]
Yeah but its compatible with my tablet so should also with phone i think (only logouts)
